eox_core/api/v1/tests/integration/test_views.py

```diff
@@ -1543,3 +1562,148 @@ def test_info_view_success(self) -> None:
         self.assertIn("version", response_data)
         self.assertIn("name", response_data)
         self.assertIn("git", response_data)
+
+
+@ddt.ddt
+# pylint: disable=too-many-ancestors
+class TestGradeAPIIntegration(
+    BaseAPIIntegrationTest,
+    UsersAPIRequestMixin,
+    EnrollmentAPIRequestMixin,
+    GradeAPIRequestMixin
+):
+    """Integration test suite for the Grade API"""
+
+    def setUp(self) -> None:
+        """Set up the test suite"""
+        super().setUp()
+        self.mode = "audit"
+        self.user = next(FAKE_USER_DATA)
+        self.create_user(self.tenant_x, self.user)
+        self.create_enrollment_data = {
+            "email": self.user['email'],
+            "course_id": self.demo_course_id,
+            "mode": self.mode,
+            "is_active": True,
+            "force": True,
+        }
+        self.params = {
+            "username": self.user['username'],
+            "course_id": self.demo_course_id,
+        }
+
+    @ddt.data(
+        {"detailed": False, "grading_policy": False},
+        {"detailed": True, "grading_policy": False},
+        {"detailed": True, "grading_policy": True},
+    )
+    def test_read_valid_params_successfully(self, extra_params: dict):
+        """
+        Get grades info from a user enrolled on a course.
```

Comment:
Should we mention that the user belongs to the tenant where the course is hosted?